src/components/Forgot/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { Form, Input, Button, Alert } from "antd";
+import firebase from "../../firebaseConfig";
+import { MailOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+
+function Forgot({ isTesting }) {
+  const { t } = useTranslation();
+  const [email, setEmail] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const forgotPassword = (e) => {
+    setErrorMessage(null);
+    firebase
+      .auth()
+      .sendPasswordResetEmail(email)
+      .then(function (user) {
+        alert("Please check your email...");
+      })
+      .catch((err) => {
+        setErrorMessage(err.message);
+      });
+  };
+
+  const layout = {
+    labelCol: { span: 4 },
+    wrapperCol: { span: 8 },
+  };
+
+  const tailLayout = {
+    wrapperCol: { offset: 8, span: 16 },
+  };
+
+  return (
+    <Form {...layout} {...tailLayout}>
+      {errorMessage ? (
```

Comment:
having this one is suffecient to render the error message line 43 is not needed

Reply:
deleted.